app/config/gojo-video/rooms.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Some weird indentation in this file mixing tabs and spaces. I'm in favor of using spaces for everything and getting rid of tabs, personally, but it would be good to fix this one way or the other.